tutorials/intermediate/constrainedlayout_guide.py

```diff
@@ -3,38 +3,46 @@
 Constrained Layout Guide
 ================================
 
-How to use constrained-layout to fit plots within your figure cleanly.
+Use *constrained layout* to fit plots within your figure cleanly.
 
-*constrained_layout* automatically adjusts subplots and decorations like
-legends and colorbars so that they fit in the figure window while still
-preserving, as best they can, the logical layout requested by the user.
+*Constrained layout* automatically adjusts subplots so that decorations like tick
+labels, legends, and colorbars do not overlap, while still preserving the
+logical layout requested by the user.
 
-*constrained_layout* is similar to
-:doc:`tight_layout</tutorials/intermediate/tight_layout_guide>`,
-but uses a constraint solver to determine the size of axes that allows
-them to fit.
+*Constrained layout* is similar to :doc:`Tight
+layout</tutorials/intermediate/tight_layout_guide>`, but is substantially more
+flexible.  It handles colorbars placed on multiple axes
```

Comment:
```suggestion
flexible.  It handles colorbars placed on multiple Axes
```
We capitalize as long as we refer to the class or logical concept of that (I.e. always unless it‘s a simple plural of axis)

Holds for multiple places here.

Reply:
Sure, changed throughout